src/_includes/base.njk

```diff
@@ -31,9 +31,8 @@
         <link href="/vendor/bootstrap/css/bootstrap.min.css" rel="stylesheet">
         <link rel="stylesheet" href="https://rsms.me/inter/inter.css">
 
-        {# Bump up version for each deploy if stylesheets have changed - temporary workaround until we sort it in the build process #}
-        <link href="/css/style.css" rel="stylesheet">
         <link href="/css/tailwind.css" rel="stylesheet">
```

Comment:
Do we still want the tailwind css?

Reply:
short answer: yes for now

Reply:
We still need Tailwind for now because I haven't updated all the places where it's used. But it's now static, rather than being built each time. Each change we make we can gradually strip out the Tailwind stuff until it's gone.